### ISSUE TITLE
Polymer paper-input + ng-form-control regression 

### ISSUE THREAD

**Issue by Zyzle** [angular/angular#5012]
Previously in alpha-44 it was possible to use paper-input with ng-form-control, in alpha-45 however this throws an error:

alpha-44 (working):
http://plnkr.co/edit/ApVnfj0RKWkgnCyaTnDT

alpha-45:
http://plnkr.co/edit/MoYgBAg6tJYb7b4aWUfS

The error:
```
Error: No value accessor for ''
    at new t (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:15:7068)
    at a (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:25:1629)
    at Object.i [as setUpControl] (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:25:1067)
    at t.onChanges (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:12:15603)
    at e.ChangeDetector_App_0.detectChangesInRecordsInternal (eval at <anonymous> (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:20214), <anonymous>:62:57)
    at e.detectChangesInRecords (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:13735)
    at e.runDetectChanges (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:13312)
    at e._detectChangesInShadowDomChildren (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:15154)
    at e.runDetectChanges (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:13425)
    at e.detectChanges (https://code.angularjs.org/2.0.0-alpha.45/angular2.min.js:20:13043)
```

**Comment by alxhub** [comment]
Looks like this was broken by 56a9b02. Previously `<paper-input>` would use the `DefaultValueAccessor`, now it doesn't get one.

A workaround is to implement your own `ControlValueAccessor`, probably extending from `DefaultValueAccessor`, with a selector that matches `paper-input`.

One short-term option would be to add an attribute such as `ng-default-control` to `DefaultValueAccessor`'s selector, to allow that directive to be applied to an arbitrary input-like element, at least until #3011 is fixed.

**Comment by alxhub** [comment]
Addressed in #5076.
